tests/testsuite/features2.rs

```diff
@@ -1047,71 +1047,6 @@ fn decouple_proc_macro() {
         .exists());
 }
 
-#[cargo_test]
-fn proc_macro_ws() {
```

Comment:
Same question about why this is being removed.

Reply:
I've brought the test back, and explicitly opted into having debuginfo generated, so that the intent of this non-regression test is preserved, without the debuginfo optimization triggering.